app/components/news-item/edit-item/component.js

```diff
@@ -26,9 +25,13 @@ export default Component.extend(DocumentsSelectorMixin, RdfaEditorMixin, {
   mandateeProposal: getCachedProperty('newsletterProposal'),
 
   themes: computed(`agendaitem.themes`, {
-    get() {
+    async get() {
       const { agendaitem } = this;
-      if (agendaitem) return A([...agendaitem.get('themes').toArray()]);
+      if (agendaitem) {
+        return await agendaitem.get('themes').then((themes) => {
+          return themes.toArray();
+        })
```

Comment:
I think this can also be written as:
```javascript
  return await agendaitem.get('themes').then(themes => themes.toArray());
```

You can choose to do this, I just know some ESLint configs will complain on this. If you decide to leave it, the upcoming ESLint will nontheless complain about a missing semicolon 🤡